<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>R2PluginsDialog</class>
 <widget class="QDialog" name="R2PluginsDialog">
  <property name="windowTitle">
   <string>radare2 plugin information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="RBinTab">
      <attribute name="title">
       <string>RBin</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="RBinLabel">
         <property name="text">
          <string>RBin plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RBinTreeWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RIOTab">
      <attribute name="title">
       <string>RIO</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="RIOLabel">
         <property name="text">
          <string>RIO plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RIOTreeWidget">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Permissions</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RCoreTab">
      <attribute name="title">
       <string>RCore</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="RCoreLabel">
         <property name="text">
          <string>RCore plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RCoreTreeWidget">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RAsmTab">
      <attribute name="title">
       <string>RAsm</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="RAsmLabel">
         <property name="text">
          <string>RAsm plugins</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="RAsmTreeWidget">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Architecture</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>CPU's</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>License</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Author</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>R2PluginsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
